front/app/containers/LandingPage/SignedOutHeader/FullWidthBannerLayout.tsx

```diff
@@ -86,11 +86,15 @@ const FullWidthBannerLayout = ({ className }: Props) => {
     const homepageSettingOpacity =
       homepageSettings.data.attributes.banner_signed_out_header_overlay_opacity;
     return (
-      <Container className={`e2e-signed-out-header ${className}`}>
+      <Container
+        data-testid="e2e-full-width-banner-layout-container"
+        className={`e2e-signed-out-header ${className}`}
```

Comment:
Could we have one of those two - either className with the e2e prefix (not preffered) or `data-cy` id?